package.json

```diff
@@ -43,7 +43,7 @@
     "jest": "^29.7.0"
   },
   "peerDependencies": {
-    "@checkdigit/eslint-plugin": "7.0.0",
+    "@checkdigit/eslint-plugin": "7.2.0",
```

Comment:
there are updates to other plugins, should we update those deps also?

Reply:
on second thoughts, if you've already tested this beta with repos, probably best to go out as-is.  lmk

Reply:
maybe we push this out first, because the other plugin deps need time to test, e.g.:

1. `@typescript-eslint/*`: 8.11.0 introduced some issue which crash rule testers, it's unclear if 8:12.0 actually fixes it or not
2. sonarjs can be a big change, and not sure if we want those new rules or not (e.g. require each file to have a test file with it, etc.)